src/components/SettingsDialog/SettingsDialog.vue

```diff
@@ -134,6 +136,7 @@ import { subscribe, unsubscribe } from '@nextcloud/event-bus'
 import MediaDevicesPreview from '../MediaDevicesPreview'
 import AppSettingsDialog from '@nextcloud/vue/dist/Components/AppSettingsDialog'
 import AppSettingsSection from '@nextcloud/vue/dist/Components/AppSettingsSection'
+import CheckboxRadioSwitch from '@nextcloud/vue/dist/Components/CheckboxRadioSwitch'
```

Comment:
> 'CheckboxRadioSwitch' is defined but never used                       

(its missing the line in the components array)

Reply:
I fixed it